### DISCUSSION TITLE
HDMI audio not working?

### DISCUSSION THREAD

**Discussion by stevannetto** [hackintosh-club/intel-nuc10#77] [General]
Hi all,
I have a NUC 10i7FNHN (the later version that comes with no integrated audio), and I used to have HDMI audio working without any issues with this EFI. However, after upgrading to Monterey and using a more recent release of the EFI folder, I don't seem to have the HDMI device available anymore. Have tried a few of the latest releases with no luck. Anyone else experiencing this issue?

**Comment by ghost** [comment]
The issue is with opencore 0.8.3 - Opencore developers changed audio code and messed it up. Use 0.8.2 for now. I'm going to report it.

**Reply by stevannetto** [reply]
Thanks, this sorted the issue.

**Comment by mrasong** [comment]
[v0.8.4.0905](https://github.com/hackintosh-efi/intel-nuc10/releases/tag/v0.8.4.0905) 
Monterey HDMI Audio now fixed.

**Reply by not-vanished** [reply]
Thanks for all the work that you do! 

I still have issues with the latest OpenCore/AppleALC to get my HDMI/USB-C audio running. Any thoughts on what could be the issue?

**Reply by mrasong** [reply]
0.8.4 HDMI audio works, but may crash.
0.8.2 works well.
